### PR TITLE
Remove Alex and add blockies sponsor

### PR DESCRIPTION
**Pull request by JasminFarhi**
- [Link to ticket](url)
- One liner ticket title/description
- If not intuitive from the ticket -> short summary of code changes
- Reccomendations for reviewers e.g. download and run code locally etc
Screen Shot 2022-11-22 at 12 09 09 pm

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -261,18 +261,24 @@ <h3 class="section-title">Find Us</h3>
...
-        <div class="affiliates col-md-6 to-animate">
+        <div class="affiliates col-md-4 to-animate">
           <h3 class="section-title">Proud Member of:</h3>
           <a href="https://blockchainaustralia.org/member/dltx/" target="_blank">
             <img src="images/Blockchain-Australia.png" width=300 />
           </a>
         </div>
+        <div class="affiliates col-md-4 to-animate">
+          <h3 class="section-title">Proud Sponsor of:</h3>
+          <a href="https://blockchainaustralia.org/member/dltx/" target="_blank">
```
Comment (OptimusOpus) [inline index.html:278]: Sonarlint wants a `rel="noopener"` on that link

**Review by bitcoinbrisbane — COMMENTED** [review]
Do we delete the jpg too?

File: index.html (hunk 1 above)
Comment (brettvniekerk) [reply index.html:278]: `rel="noreferrer noopener"` pls

Comment (JasminFarhi) [reply index.html:278]: done

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**